# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chrony/chrony_devfs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naur chrony-1.20.orig/conf.c chrony-1.20/conf.c
--- chrony-1.20.orig/conf.c	2003-09-22 22:22:30.000000000 +0100
+++ chrony-1.20/conf.c	2003-10-04 22:43:53.000000000 +0100
@@ -96,7 +96,7 @@
 /* ================================================== */
 /* Configuration variables */
 
-static char *rtc_device = "/dev/rtc";
+static char *rtc_device = "/dev/misc/rtc";
 static int acquisition_port = 0; /* 0 means let kernel choose port */
 static int ntp_port = 123;
 static char *keys_file = NULL;
